### ISSUE TITLE
Exception only logs to Sentry UI after rerunning script

### ISSUE THREAD

**Issue by aliwoodman** [getsentry/sentry-python#843]
Hi, 

I'm working on an app which has a bunch of python scripts. We've recently added Sentry logging to one of these, following the guide https://docs.sentry.io/platforms/python/ . Here's a simplified example of the script:

```
import sentry_sdk
from meya import Component

class exampleComponent(Component):

    def start(self):
            sentry_sdk.init(dsn='insert_real_dsn')

            try:
                raise Exception('Test error')
            except Exception as e:
                sentry_sdk.capture_exception(e)

```

With the platform we're using we don't have access to a global entry point, so we init Sentry in the script. 

When we test raising an exception, we do find that this gets logged to Sentry, but only after running the script for a second time. 

If we run the script a third time, the error captured on the second run gets logged to Sentry, and so on.

Is this expected behaviour? If so, can you explain why it is the case? If not, is there an obvious reason why we might be seeing this behaviour?

Thanks!

**Comment by untitaker** [comment]
Hey, there's no reason this would happen, the SDK does not store local state between runs. It may just be that the error takes that long to get saved on the servers. Sentry takes a couple of seconds to save the error after the upload has finished.

**Comment by aliwoodman** [comment]
It's definitely not just a few seconds delayed unfortunately - I tested running the script, waiting for 5 mins checking the UI, no error, running the script again, the first error came through. In the real version I've been able to confirm I'm referencing the right errors by printing out and logging a randomly generated error each time.

I forgot to mention that it sounds like something which has been encountered for the java sdk e.g. https://github.com/getsentry/sentry-java/issues/916. Could something similar be happening in the python version? My script stops running after executing - is it possibly no long enough for the error to make its way all the way through the system?
